templates/subjqa/restaurants/templates.yaml

```diff
@@ -0,0 +1,137 @@
+dataset: subjqa
+subset: books
+templates:
+  071f2b19-7392-4258-8a60-5a96f3e44b0d: !Template
+    id: 071f2b19-7392-4258-8a60-5a96f3e44b0d
+    jinja: 'Context:
+
+      {{context}}
+
+
+      Answer the following question with extracts from the context: {{question}}
+
+
+      |||
+
+      {% if (answers["text"]  | length) == 0 %}
+
+      {{ "Unanswerable" }}
+
+      {% else %}
+
+      {{answers["text"] | join(" \n ")}}
+
+      {% endif %}'
+    name: answer_q_with_context_first
+    reference: ''
+    task_template: true
+  a217525b-caf2-4ae3-8a6e-06bd48bf4728: !Template
+    id: a217525b-caf2-4ae3-8a6e-06bd48bf4728
+    jinja: '{{question}}
+
+
+      Answer using extracts from the following context. If you can''t find an answer,
+      return {{"Unanswerable"}}
+
+
+      Context:
+
+      {{context}}
+
+
+      |||
+
+      {% if (answers["text"]  | length) == 0 %}
+
+      {{ "Unanswerable" }}
+
+      {% else %}
+
+      {{answers["text"] | join(" \n ")}}
+
+      {% endif %}'
+    name: answer_q_with_context_after
+    reference: ''
+    task_template: true
+  afe5086e-d9fe-4981-bcac-67d580950110: !Template
+    id: afe5086e-d9fe-4981-bcac-67d580950110
+    jinja: 'Question:
+
+      {{question}}
+
+
+      What is the subjectivity score of the question (on a 1 to 5 scale with 1 being
```

Comment:
Reformulation for diversity: `On a scale of 1 to 5 (1 being the most subjective), how subjective is the question?`

Reply:
Done on all subsets